### PR TITLE
Add k6chaijs 4.5.0.0 - dropping need for 'global' alias

### PR DESCRIPTION
**Pull request by mstoykov**
## Description

Update to latest version of k6chaijs

## Please fill in this template.

- [ ] Use a meaningful title for the Pull Request. Include the name of the jslib added/modified.
- [ ] Fill the description section of the Pull Request. 
- [ ] Test the change in your code, and ensure the `npm run test` command succeeds.
- [ ] Run `yarn run generate-homepage` locally and verify the new homepage `/lib/index.html` file looks legit.

**Select one of these and delete the others**:

*If adding a new jslib*:
- [ ] The Pull Request creates a `/lib/{jslib_name}` folder.
- [ ] The Pull Request creates a `/lib/{jslib_name}/{desired_version}` folder.
- [ ] The `/lib/{jslib_name}/{desired_version}/index.js` file containing the jslib's code bundle exists.
- [ ] The Pull Request updates the `supported.json` file to contain an entry for the newly added jslib and its `{desired_version}`, as in the following example:
```JSON
{
  "{jslib_name}": {
    // Available package versions
    "versions": [
      "{desired_version}"
    ],

    // (optional) Documentation's or repository's URL
    "docs-url": "{documentation_or_repository_url}",

    // (optional) As a default, the homepage will point to
    // a package's bundle's index.js. If your package's main
    // bundle name is different; set it here (see the AWS
    // package for instance).
    "bundle-filename": "{index.js}"
}
```
- [ ] Tests have been added to `/tests/basic.js` and `/tests/testSuite.js` to ensure that the added jslib is importable and runnable by k6.

*If publishing a new version of an existing jslib*:
- [ ] The Pull Request is labeled with the `version bump` label.
- [ ] The Pull Request adds a `/lib/{jslib_name}/{desired_version}` folder.
- [ ] The Pull Request adds a `/lib/{jslib_name}/{desired_version}/index.js` file containing the jslib's code bundle.
- [ ] The Pull Request updates the `supported.json` file to contain an entry for the newly added jslib version, as in the following example:
```JSON
{
  "my-lib": [
    "1.0.2",
    // Use semantic versioning
    "{desired_version}"
  ]
}
```
- [ ] The Pull Request adds the relevant tests to the `/tests/basic.js` and `/tests/testSuite.js` files to ensure that the new version of the jslib is importable and runnable by k6.
- Merge the Pull Request once it is green. PRs adding new jslib versions do not require to get a review to be merged :rocket:.

### CODE REVIEW

**Review by codebien — APPROVED** [review]

File: supported.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "docs-url": "https://k6.io/docs/javascript-api/jslib/expect"
   },
   "k6chaijs": {
-    "versions": ["4.3.4.0", "4.3.4.1", "4.3.4.2", "4.3.4.3"],
+    "versions": ["4.3.4.0", "4.3.4.1", "4.3.4.2", "4.3.4.3", "4.5.0"],
```
Comment (joanlopez) [inline supported.json:44]:
```suggestion
    "versions": ["4.3.4.0", "4.3.4.1", "4.3.4.2", "4.3.4.3", "4.5.0.0"],
```

File: lib/index.html [hunk 2]
```diff
@@ -612,7 +612,7 @@ <h2>Available libs</h2>
       <td><a href="https://k6.io/docs/javascript-api/jslib/expect">https://k6.io/docs/javascript-api/jslib/expect</a></td>
     </tr><tr>
       <td>k6chaijs</td>
-      <td><a target="_blank" href="https://jslib.k6.io/k6chaijs/4.3.4.0/index.js">4.3.4.0</a>, <a target="_blank" href="https://jslib.k6.io/k6chaijs/4.3.4.1/index.js">4.3.4.1</a>, <a target="_blank" href="https://jslib.k6.io/k6chaijs/4.3.4.2/index.js">4.3.4.2</a>, <a target="_blank" href="https://jslib.k6.io/k6chaijs/4.3.4.3/index.js">4.3.4.3</a></td>
+      <td><a target="_blank" href="https://jslib.k6.io/k6chaijs/4.3.4.0/index.js">4.3.4.0</a>, <a target="_blank" href="https://jslib.k6.io/k6chaijs/4.3.4.1/index.js">4.3.4.1</a>, <a target="_blank" href="https://jslib.k6.io/k6chaijs/4.3.4.2/index.js">4.3.4.2</a>, <a target="_blank" href="https://jslib.k6.io/k6chaijs/4.3.4.3/index.js">4.3.4.3</a>, <a target="_blank" href="https://jslib.k6.io/k6chaijs/4.5.0/index.js">4.5.0</a></td>
```
Comment (joanlopez) [inline lib/index.html:615]:
```suggestion
      <td><a target="_blank" href="https://jslib.k6.io/k6chaijs/4.3.4.0/index.js">4.3.4.0</a>, <a target="_blank" href="https://jslib.k6.io/k6chaijs/4.3.4.1/index.js">4.3.4.1</a>, <a target="_blank" href="https://jslib.k6.io/k6chaijs/4.3.4.2/index.js">4.3.4.2</a>, <a target="_blank" href="https://jslib.k6.io/k6chaijs/4.3.4.3/index.js">4.3.4.3</a>, <a target="_blank" href="https://jslib.k6.io/k6chaijs/4.5.0.0/index.js">4.5.0.0</a></td>
```

**Review by joanlopez — COMMENTED** [review]
Looks like there is an inconsistency in the version format, sometimes it is referred as `4.5.0` while sometimes it is as `4.5.0.0`, and beyond possible confusion that may cause, I think the import/links won't work if they aren't consistent among them.

**Review by 2Steaks — APPROVED** [review]
LGTM based on @joanlopez comments

**Review by joanlopez — APPROVED** [review]

**Pull request merged**